podcasts/AuthenticationHelper.swift

```diff
@@ -67,7 +74,9 @@ class AuthenticationHelper {
             ServerSettings.setSyncingEmail(email: response.email)
         }
 
-        NotificationCenter.default.post(name: .userLoginDidChange, object: nil)
+        DispatchQueue.main.async {
+            NotificationCenter.default.post(name: .userLoginDidChange, object: nil)
+        }
```

Comment:
There's a helper extension that handles this. 
```suggestion
        NotificationCenter.postOnMainThread(notification: .userLoginDidChange)
```